packages/serverless-contracts/src/types/genericContract.ts

```diff
@@ -1,6 +1,9 @@
 import { JSONSchema } from 'json-schema-to-ts';
 
+import { ContractOpenApiDocumentation } from './contractOpenApiDocumentation';
+
 export interface GenericContract {
-  get fullContractSchema(): JSONSchema;
   get contractId(): string;
+  get fullContractSchema(): JSONSchema;
```

Comment:
To be honest, I think this interface should remain very generic. Not all contracts can have an openApi documentation. Maybe a different `OpenApiGenericContract` extending this one would do